install/kubernetes/helm/subcharts/kiali/templates/ingress.yaml

```diff
@@ -15,17 +15,26 @@ metadata:
     {{- end }}
 spec:
   rules:
+{{- if .Values.ingress.hosts }}
     {{- range $host := .Values.ingress.hosts }}
     - host: {{ $host }}
       http:
         paths:
-          - path: /
+          - path: {{ if $.Values.contextPath }} {{ $.Values.contextPath }} {{ else }} / {{ end }}
             backend:
               serviceName: kiali
               servicePort: 20001
     {{- end -}}
+{{- else }}
+    - http:
+        paths:
+          - path: {{ if .Values.contextPath }} {{ .Values.contextPath }} {{ else }} / {{ end }}
+            backend:
+              serviceName: kiali
+              servicePort: 20001
+{{- end }}
   {{- if .Values.ingress.tls }}
   tls:
 {{ toYaml .Values.ingress.tls | indent 4 }}
   {{- end -}}
-{{- end -}}
+{{- end -}}
```

Comment:
remove this

![image](https://user-images.githubusercontent.com/4461983/47330308-572ab080-d6aa-11e8-8b0e-b3fe14f9ab25.png)